<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>621</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Действия при повторении имени выходного файла</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="checkBox_delete_yes">
       <property name="text">
        <string>Да</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_perem"/>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="checkBox_timer_no">
       <property name="text">
        <string>Единоразово</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Режим работы</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_save"/>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Выбор файла для модификации(пожалуйста, заполните строку &quot;маска файла&quot; перед выбором)</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_timer"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Удалять ли входные файлы?</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QCheckBox" name="checkBox_timer_yes">
       <property name="text">
        <string>По таймеру</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Значение для модификации</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Путь для сохранения файлов</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Маска файла</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QCheckBox" name="checkBox_delete_no">
       <property name="text">
        <string>Нет</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_mask"/>
     </item>
     <item row="6" column="2">
      <widget class="QCheckBox" name="checkBox_overwrite_no">
       <property name="text">
        <string>Модификация файла</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Периодичность работы таймера(в секундах)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="checkBox_overwrite_yes">
       <property name="text">
        <string>Перезапись</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Начать работу программы(при единоразовом запуске)</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1" colspan="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Пуск таймера(при работе по таймеру)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выполнение операции XOR над содержимым файла</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
